WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -596,3 +597,243 @@ extension ReaderPostCardCell: ReaderCardDiscoverAttributionViewDelegate {
         delegate?.readerCell(self, attributionActionForProvider: contentProvider!)
     }
 }
+
+extension ReaderPostCardCell: Accessible {
+    func prepareForVoiceOver() {
+        prepareCardForVoiceOver()
+        prepareHeaderButtonForVoiceOver()
+        prepareShareForVoiceOver()
+        prepareCommentsForVoiceOver()
+        prepareLikeForVoiceOver()
+        prepareMenuForVoiceOver()
+        prepareVisitForVoiceOver()
+        prepareFollowButtonForVoiceOver()
+    }
+
+    private func prepareCardForVoiceOver() {
+        accessibilityLabel = cardAccessibilityLabel()
+        accessibilityHint = cardAccessibilityHint()
+        accessibilityTraits = UIAccessibilityTraitButton
+    }
+
+    private func cardAccessibilityLabel() -> String {
+        let authorName = postAuthor()
+        let blogTitle = blogName()
+
+        return headerButtonAccessibilityLabel(name: authorName, title: blogTitle) + "," + postTitle() + "," + postContent()
```

Comment:
I don't think it really matters, as I don't think this text will ever be _displayed_ anywhere, but should these commas have spaces after them?  Same in `headerButtonAccessibilityLabel` below.